src/js/apps/patients/list/patients-all_app.js

```diff
@@ -25,6 +25,8 @@ export default App.extend({
     const groupId = this.getState('groupId') || groups.at(0).id;
     this.setState({ groupId });
 
+    if (groups.length === 1) return;
```

Comment:
can we add a bail like this for `0` above 25 as a cheap work around no groups?  I'm not sure what the endpoint does when no group filter is sent, but I imagine it'll either be 0 results or something that just leaves the spinner vs a console error.

Reply:
actually, let's try it and if it's no results, this is probably worth doing.. but if it's a spinner, no groups should just be handled differently.

Reply:
the endpoint 400's, so we'll probably need to handle it differently right?

Reply:
yep ok